### ISSUE TITLE
Minor bug in Gaussian Diffusion implementation (q_mean_var)?

### ISSUE THREAD

**Issue by SeminKim** [tqch/ddpm-torch#9]
Hi, thanks for sharing your code.
I found that `q_mean_var` function in `GaussianDiffusion` implementation is strange.
https://github.com/tqch/ddpm-torch/blob/9a8ecb57cf1ab2dca04ba1135b09992d456108a8/ddpm_torch/diffusion.py#L86-L90
I think those lines have bug; the `logvar` variable should literally store log value of var (like `torch.log(var)`) maybe?
If I am misunderstanding something, I would appreciate your help.
Thanks.

**Comment by tqch** [comment]
Hey @SeminKim , good catch indeed! You're absolutely right about the log-variance calculation. It should be $\log(1-\bar{\alpha}_t)$ instead. I have corrected it in 9a8ecb5. That being that, since `q_mean_var` is only used to calculate `_prior_bpd` which is the `L_T` in the original paper and this term actually contains no learnable parameters, the typo should not have any effect on the models trained by previous code. Besides, it is unused when using the simple loss ($\ell_2$ loss).

**Comment by SeminKim** [comment]
Yes, I agree that this would not harm any of previous trained models. Sounds clear to me, thanks for the prompt reply:)
